requirements.txt

```diff
@@ -0,0 +1,13 @@
+absl-py>=0.7.1
+appdirs>=1.4.3
+gast>=0.2.2
+h5py>=2.9.0
+matplotlib==3.3.2
+numpy>=1.16.2
+oauthlib>=2.1.0
+Pillow>=5.4.1
+PyYAML>=5.1.2
+requests>=2.25
```

Comment:
@bhavikapanara do we need `requests, requests-oauthlib, responses, simplejson, oauthlib, matplotlib, h5py, gast, appdirs, absl-py` ?

Please do a thorough cleanup to ensure there are no unused packages in requirements.txt.

Reply:
done